beekeeper-security-plugin/build.gradle

```
@@ -1,6 +1,6 @@
 dependencies {
     implementation 'org.owasp:dependency-check-gradle:5.3.2.1'
-    implementation 'org.cyclonedx:cyclonedx-gradle-plugin:1.4.0'
+    implementation 'org.cyclonedx:cyclonedx-gradle-plugin:1.5.0'
```

```suggestion
    implementation 'com.cyclonedx:cyclonedx-gradle-plugin:1.5.0'
```

Seems like the latest release is only available under `com.cyclonedx` and not `org.cyclonedx`.